docker-compose/7.1.N-docker-compose.yml

```diff
@@ -59,6 +59,16 @@ services:
         -XX:MinRAMPercentage=50
         -XX:MaxRAMPercentage=80
         -XX:MaxRAM=1900m
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.alfresco.rule=PathPrefix(`/`)"
+      - "traefik.http.services.alfresco.loadbalancer.server.port=8080"
+      - "traefik.http.routers.solrapideny.rule=PathRegexp(`^/alfresco/(wc)?s(ervice)?/api/solr/`)"
+      - "traefik.http.middlewares.fakeauth.basicauth.users=test:"
+      - "traefik.http.routers.solrapideny.middlewares=fakeauth@docker"
+      - "traefik.http.routers.alfrescomicrometer.rule=PathRegexp(`^/alfresco/(wc)?s(ervice)?/prometheus`)"
+      - "traefik.http.middlewares.prometheusipfilter.ipallowlist.sourcerange=127.0.0.0/8"
+      - "traefik.http.routers.alfrescomicrometer.middlewares=prometheusipfilter@docker"
```

Comment:
chatgpt suggests pointing the middleware to a non-existing upstream, plus setting the status explicitly to 403:

```
    labels:
      - "traefik.http.routers.solr-block-router.rule=PathRegexp(`/.*?/service/api/solr/.*`) || PathRegexp(`/.*?/s/api/solr/.*`) || PathRegexp(`/.*?/wcservice/api/solr/.*`) || PathRegexp(`/.*?/wcs/api/solr/.*`) || PathRegexp(`/.*?/proxy/.*/api/solr/.*`) || PathRegexp(`/.*?/-default-/proxy/.*/api/.*`)"
      - "traefik.http.routers.solr-block-router.entrypoints=web"
      - "traefik.http.routers.solr-block-router.middlewares=solr-block-middleware"
      - "traefik.http.middlewares.solr-block-middleware.errorpage.status=403"
      - "traefik.http.middlewares.solr-block-middleware.errorpage.service=noop-service"
      - "traefik.http.middlewares.solr-block-middleware.errorpage.query=/"
      - "traefik.http.services.noop-service.loadbalancer.server.url=http://0.0.0.0"
```

which may be more explicit?

Reply:
both are kinda hacky. I think the one based on `authBasic` middleware is easier to read.
Also I tried and it seems to conflict with the existing router rule as I get errors like:
```
Router alfresco cannot be linked automatically with multiple Services: ["alfresco" "noop-service"] providerName=docker routerName=alfresco
```